dev-docs/internal-api-reference/registerBidAdapter.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: internal_api_prebidjs
+title: pbjs.registerBidAdapter(bidderAdapter, bidderCode)
+description: Wrapper to register a bid adapter externally
+sidebarType: 1
+---
+
+This function provides a wrapper to register a bid adapter externally.
```

Comment:
what does "registering externally" mean? i.e. what happens if this function is not called?

Reply:
So this is how an Amazon adapter was able to be added without being in the project. You can create an adapter and then load it through this function without the adapter living in the project itself